### DISCUSSION TITLE
cam I use my external monitor as only display for SEB

### DISCUSSION THREAD

**Discussion by aurz1599** [SafeExamBrowser/seb-win-refactoring#259] [Q&A] [answered]
I'm having some trouble as Prohibited Display Configuration when starting SEB. I know that it is not permitted to use 1 internal and 1 external display (for my teacher setting). However, **I see that 1 external display is allowed.** So, if it possible that I turn off my internal display (notebook) and use only an external display (my external monitor). I tried to use the window setting “show only 2 (2 is my external display)”, and exam.net is not allowed. Thanks!

**Comment by dbuechel** [comment, marked as answer]
If there's only one display allowed in the configuration, then you'll only be able to use one display. In case of a laptop with an integrated display, you won't be able to only use the external display as you can't disconnect the internal one.
